<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>355</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>330</width>
      <height>400</height>
     </rect>
    </property>
    <widget class="QWidget" name="widgetTeamScore6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelScore6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Malgun Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumber6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>40</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelTeamName6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>喝奶茶不吐奶茶皮</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarScore6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetTeamScore2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelScore2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Malgun Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumber2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>40</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelTeamName2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>喝奶茶不吐奶茶皮</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarScore2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetTeamScore5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelScore5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Malgun Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumber5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>40</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelTeamName5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>喝奶茶不吐奶茶皮</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarScore5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetTeamScore1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelScore1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Malgun Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumber1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>45</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelTeamName1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>喝奶茶不吐奶茶皮</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarScore1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetTeamScore3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelScore3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Malgun Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumber3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>40</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelTeamName3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>喝奶茶不吐奶茶皮</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarScore3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetTeamScore4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="labelScore4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Malgun Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelNumber4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Sans</family>
        <pointsize>40</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelTeamName4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>喝奶茶不吐奶茶皮</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarScore4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>25</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelScoreBoard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold Condensed</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SCORE BOARD</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="widgetBottom" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonRevoke">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>←</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTimeLength">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditIncrement">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>330</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>2:00</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold Condensed</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TIMER</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
